tidbctl/command/table_id_command.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"fmt"
+
+	"encoding/json"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"net/http"
+)
+
+type tableInfo struct {
+	Name string `json:"name"`
+	ID   int64  `json:"id"`
+}
+
+func (t tableInfo) String() string {
+	return fmt.Sprintf("tableInfo{name: %s, id: %d}", t.Name, t.ID)
+}
+
+// NewTableIDCommand return the tableID of the given tableName
+func NewTableIDCommand() *cobra.Command {
+	m := &cobra.Command{
+		Use:   "showTableID [flags] <tidb_addr> <db_name> <table_name>",
+		Short: "show the table id given the table name",
+		Run:   showTableIDCommandFunc,
+	}
+
+	// Add flags for command `showTableID`
+	m.Flags().BoolP("verbose", "v", false, "Print tableInfo in verbose mode")
+
+	return m
+}
+
+func showTableIDCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 3 {
+		fmt.Println("Usage: showTableID <tidb_addr> <db_name> <table_name>")
+		return
+
+	}
+	host := args[0]
```

Comment:
I think we must pass TiDB address to the tidb-ctl at first, not in the sub command.